integration_tests/test_mempool.py

```
@@ -115,10 +115,26 @@ def test_mempool_nonce(cronos_mempool):
+    # drain any pending txs from prior tests so we start with clean nonce state
+    max_drain_blocks = 20
+    for _ in range(max_drain_blocks):
+        if w3.eth.get_transaction_count(
+            sender, "pending"
+        ) == w3.eth.get_transaction_count(sender):
+            break
+        wait_for_new_blocks(cli, 1, sleep=0.1)
+    else:
+        pending = w3.eth.get_transaction_count(sender, "pending")
+        latest = w3.eth.get_transaction_count(sender)
+        assert False, (
```

Ruff (0.15.13)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)